packages/core/src/Stores/client-store.js

```diff
@@ -2718,6 +2720,28 @@ export default class ClientStore extends BaseStore {
         this.should_show_effortless_login_modal = should_show_effortless_login_modal;
     }
 
+    setPasskeysStatusToCookie(status) {
+        let domain = /deriv.com/.test(window.location.hostname) ? URLConstants.derivHost : window.location.hostname;
+
+        if (/deriv.dev/.test(window.location.hostname)) {
+            //set domain for dev environment (FE deployment and login page on qa-box)
+            domain = 'deriv.dev';
+        }
+
+        const expirationDate = new Date();
+        expirationDate.setFullYear(expirationDate.getFullYear() + 1); // Set to expire in 1 year
```

Comment:
did we confirm the expiry for this cookie value?

Reply:
@amina-deriv, we just discussed it during passkeys meetings during the week.
basically, there is no expiration time. we just need to store that flag. we will rewrite it anytime we create passkey.
there is no sensitive data, so no issue here we believe